presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt

```diff
@@ -1,126 +1,286 @@
+#$ DESCRIPTION:*[DISCORD SUPPORT/FEEDBACK](https://discord.gg/Gs5nTx4rE2)
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION: 
+
 #$ INCLUDE: presets/4.3/tune/defaults.txt
-#$ INCLUDE: presets/4.3/filters/defaults.txt
 
-# -- Non-RPM Filtering --
+# -- Antigravity --
+
+set anti_gravity_gain = 5000
+
+#$ OPTION_GROUP BEGIN:PIDs
+
+    #$ OPTION BEGIN (CHECKED): 4s Lipo with HD Camera (2400-2700kv motors)
+
+        set simplified_master_multiplier = 130
+        set simplified_i_gain = 85
+        set simplified_d_gain = 110
+        set simplified_pi_gain = 110
+        set simplified_dmax_gain = 0
+        set simplified_feedforward_gain = 95
+        set simplified_pitch_pi_gain = 115
+
+    #$ OPTION END
+
+    #$ OPTION BEGIN (UNCHECKED):4s Lipo no HD Camera (2400-2700kv motors)
+
+        set simplified_master_multiplier = 130
+        set simplified_i_gain = 85
+        set simplified_d_gain = 110
+        set simplified_pi_gain = 110
+        set simplified_dmax_gain = 0
+        set simplified_feedforward_gain = 95
+        set simplified_pitch_pi_gain = 105
+
+    #$ OPTION END
+
+    #$ OPTION BEGIN (UNCHECKED):6s Lipo with HD Camera (1750 - 1950kv motors)
+
+        set simplified_master_multiplier = 115
+        set simplified_i_gain = 85
+        set simplified_d_gain = 110
+        set simplified_pi_gain = 110
+        set simplified_dmax_gain = 0
+        set simplified_feedforward_gain = 95
+        set simplified_pitch_pi_gain = 115
+
+    #$ OPTION END
+
+    #$ OPTION BEGIN (UNCHECKED):6s Lipo no HD Camera (1750 - 1950kv motors)
+
+        set simplified_master_multiplier = 115
+        set simplified_i_gain = 85
+        set simplified_d_gain = 110
+        set simplified_pi_gain = 110
+        set simplified_dmax_gain = 0
+        set simplified_feedforward_gain = 95
+        set simplified_pitch_pi_gain = 105
+
+    #$ OPTION END
+
+#$ OPTION_GROUP END
+
+# ---- Filters Base Setup -------
+
+#$ INCLUDE: presets/4.3/filters/defaults.txt
 
 set dshot_bidir = OFF
-set simplified_gyro_filter = OFF
-set gyro_lpf2_static_hz = 1000
-set gyro_lpf1_dyn_min_hz = 300
-set gyro_lpf1_dyn_max_hz = 600
-set dyn_notch_count = 3
-set dyn_notch_q = 300
-set dyn_notch_min_hz = 100
+set simplified_gyro_filter_multiplier = 120
+set gyro_lpf1_static_hz = 0
+set gyro_lpf1_dyn_min_hz = 0
+set gyro_lpf1_dyn_max_hz = 0
+set dyn_notch_count = 5
+set dyn_notch_q = 400
+set dyn_notch_min_hz = 90
 set dyn_notch_max_hz = 800
 
-# -- Dterm filtering --
-
-set simplified_dterm_filter = OFF
+set simplified_dterm_filter_multiplier = 140
 set dterm_lpf1_dyn_min_hz = 0
 set dterm_lpf1_dyn_max_hz = 0
 set dterm_lpf1_dyn_expo = 0
-set dterm_lpf1_static_hz = 80
-set dterm_lpf2_static_hz = 140
 
-# -- Antigravity --
+#$ OPTION_GROUP BEGIN: Filters (default to MPU6000/BMI/ICM206xx Gyro)
 
-set anti_gravity_gain = 4000
+    #$ OPTION BEGIN (CHECKED): RPM filtering (Uncheck for Dynamic Notch Filter Setup)
+    #$ INCLUDE_WARNING: misc/warnings/en/rpm_filters.txt
```

Comment:
warnings don't work like that, unfortunately. Even tho they can be placed anywhere in the code, they will popup regardless of anything. So its better to place all the warnings at the header of the preset, before long description

Reply:
...so I'm ahead of the game in this position right? ;)

Shall I change it if it doesnt make a difference currently?

Reply:
Can you check if it even works if its inside the option? :)

Reply:
> Can you check if it even works if its inside the option? :)

I moved them out now - pls check new top lines...guess I should also on yesterdays 3/4 Inch..

Reply:
yeah if this slipped through better change that... or at least check that warnings are working